esmvaltool/_recipe.py

```diff
@@ -451,6 +451,17 @@ def _get_default_settings(variable, config_user, derive=False):
     settings['save'] = {
         'compress': config_user['compress_netcdf']}
 
+    # Configure ingestion of landocean masks
```

Comment:
This should probably not be here, only preprocessor functions that are applied by default should go in this function, hence the name  `_get_default_settings`. Now the preprocessor will always try to apply land/ocean masking if the 'sftlf' mask is specified in the variable section of the recipe.